esp/logging/loggingmanager/loggingmanager.cpp

```diff
@@ -123,7 +123,8 @@ bool CLoggingManager::updateLog(const char* option, IEspContext& espContext, IPr
         const char* userId = espContext.queryUserId();
         if (userId && *userId)
             espContextTree->addProp("UserName", userId);
-        espContextTree->addPropInt64("ResponseTime", (__int64)(msTick()-espContext.queryCreationTime())/1000);
+
+        espContextTree->addProp("ResponseTime", VStringBuffer("%.4f", (msTick()-espContext.queryCreationTime())/1000.0));
```

Comment:
You may need to add .str() after the VStringBuffer to match: void addProp(const char *xpath, const char *val).